test/libsolidity/syntaxTests/more_than_256_warnings.sol

```diff
@@ -0,0 +1,280 @@
+// Warning 5805: (2543-2547): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2553-2557): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2563-2567): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2573-2577): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2583-2587): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2601-2605): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2611-2615): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2621-2625): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2631-2635): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2641-2645): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2651-2655): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2661-2665): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2671-2675): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2681-2685): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2691-2695): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2701-2705): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2711-2715): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2721-2725): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 5805: (2731-2735): "this" used in constructor. Note that external functions of a contract cannot be called while it is being constructed.
+// Warning 4591: There are more than 256 warnings. Ignoring the rest.
```

Comment:
Don't we have such a test already? I added one years ago, maybe only for errors?

Reply:
Indeed, there are several `syntaxTest/more_than_256_*.sol` files, but they are for errors.

Reply:
The mechanism is that it starts ignoring warnings, but not errors. Can you add something to the code that generates an error after all the warnings?

Reply:
Do you mean to check that we are still reporting errors after 256 warnings? Sure, we can even have a single test for 256 warnings and 256 errors.

But it would not fix the initial problem that I mentioned. The number of warnings is different for different compiler builds, in particular because of `"This is a pre-release compiler version, please do not use it in production."` warning.

One possible option would be to suppress certain warnings via command line arguments, but that might be overdoing. Another option would be just to white-list the "256+"-warning, and ignore it for coverage.

Reply:
Then let's write a different test and ignore the number for coverage.